tests/kafkatest/tests/consumer_group_command_test.py

```diff
@@ -80,7 +80,7 @@ def setup_and_verify(self, security_protocol, group=None):
             command_config_file = self.COMMAND_CONFIG_FILE
 
         if group:
-            wait_until(lambda: ("%s, topic-consumer-group-command, 0," % group) in self.kafka.describe_consumer_group(group=group, node=kafka_node, new_consumer=enable_new_consumer, command_config=command_config_file), timeout_sec=10,
+            wait_until(lambda: ("%s            topic-consumer-group-command   0" % group) in self.kafka.describe_consumer_group(group=group, node=kafka_node, new_consumer=enable_new_consumer, command_config=command_config_file), timeout_sec=10,
```

Comment:
This seems pretty brittle -- it's totally tied to spacing choices made in the tool. At a minimum `setup_and_verify` isn't actually generic -- it depends on the length of `group` if `group` is not `None`. Actually, I'm not even clear on why the original PR changed the formatting? I get that the formatting might be nicer for display, but it seems a lot harder to parse, and I know some other tools keep things parseable (e.g. perf tools). On the other hand, others seem to prefer optimizing for human readability (e.g. kafka-topics).

There's a JIRA somewhere for providing something like `--json` options across all tools, and maybe that's the best long term solution, but this fix seems likely to break if anyone tweaks the output, so maybe just converting to a regex to match the relevant whitespace will be good enough for now?